features-sdk/run.sh

```diff
@@ -8,5 +8,5 @@ export SCOPE=${1:-features-sdk}
 
 for protocol in websocket http; do
   # profiles are defined in the cucumber.js file at the root of this project
-  KUZZLE_PROTOCOL=$protocol ./node_modules/.bin/cucumber-js --profile $protocol $SCOPE
+  KUZZLE_PROTOCOL=$protocol npx cucumber-js --profile $protocol $SCOPE
```

Comment:
Why switching to `npx` now?

Reply:
All it does is to fetch module binaries in the right place, this prevents us to type the exact path by ourselves. I find this a bit cleaner.

Reply:
Ok great 👍